### DISCUSSION TITLE
solucion sencilla por lo general es mejor

### DISCUSSION THREAD

**Discussion by rglvlncsr** [atralice/Curso.Prep.Henry#41] [Ideas]
function esEntero(numero) {
  // Devuelve "true" si "numero" es un entero (int/integer)
  // Ejemplo: 0.8 -> false
  // Ejemplo: 1 -> true
  // Ejemplo: -10 -> true
  // De lo contrario, devuelve "false"
    // Pista: Puedes resolver esto usando `Math.floor`
    // codigo de henry : return numero % 1 === 0;
    // Tu código:
    //   return numero % 1 === 0;

    if (numero, Math.floor(numero) % 1 ===0) {
       return (true);
    } else {
        return (false);
    }

}
    

el asunto es que se me pide diferenciar un entero y un decimal con Math.floor y me vuelve loco(parece que volvi a primaria)porque miren ese codigo que hice...
la respuesta es mas sencilla es como ustedes la proponen:
return numero % 1 === 0;

asi la puse y para ser practico esta bien pero....como lo hago com Math.floor....no quiero la respuesta solo una pequena orientacion.
He buscado en la red pero infructuosamente...

**Comment by atralice** [comment]
Buenas,
Ponele que tenes un número cualquiera: `1.2`
Si yo le hago un `Math.floor` entonces da `1`.
Si comparo el numero original con el numero pasado por el `Math.floor` puedo saber si tenía una parte real o no. Se ve?

**Comment by ninosoto** [comment]
Si se ve claramente. Gracias
